peer/peer.go

```diff
@@ -811,39 +811,38 @@ func (p *Peer) pushVersionMsg() error {
 }
 
 // PushAddrMsg sends an addr message to the connected peer using the provided
-// addresses.  This function is useful over manually sending the message via
+// addresses. This function is useful over manually sending the message via
```

Comment:
This doesn't really matter, but if you look throughout the project there are pretty much always two spaces after a period.  I'd prefer to keep it that way for consistency.